core/src/main/kotlin/ch/ergon/dope/resolvable/clause/ISelectClause.kt

```diff
@@ -28,32 +28,35 @@ import ch.ergon.dope.validtype.NumberType
 import ch.ergon.dope.validtype.StringType
 import ch.ergon.dope.validtype.ValidType
 
-interface ISelectOffsetClause : Clause
+interface ISelectOffsetClause<T : ValidType> : Clause, TypeExpression<ArrayType<T>> {
```

Comment:
What is with them?

Reply:
If you mean to add the TypeExpression to delete and update then i can tell you that thats not possible, because you can't create a Subquery for delete or update in a SelectClause.